### DISCUSSION TITLE
Help Needed with Customizations and Settings

### DISCUSSION THREAD

**Discussion by 0x1holon** [prasanthrangan/hyprdots#1774] [Q&A]
Hi Community
I hope this message finds you well. I’m new to Arch Linux and currently working with your project. I have a few questions and would greatly appreciate any assistance you could provide:
    Changing Size of Waybar and Applications: How can I change the size of Waybar? Additionally, I would like to know how to adjust the size settings for applications like Firefox.
    Improving Monitor Refresh Rate: My second monitor has a relatively low refresh rate. Could someone guide me on how to improve or fix this issue?
    Keyboard Layout and Bindings: I need to configure both US and RU keyboard layouts and add custom key bindings. I tried to change this through the hyprland config but was unsuccessful. Could you provide guidance on the correct approach or steps to follow?
    Binding Space to Alt Gr: Is there a way to bind the Space key to function as Alt Gr? I would appreciate any guidance on how to set this up.
Thank you in advance for your help and understanding as I navigate these configurations!
Best regards,
holooon

**Comment by kRHYME7** [comment]
>  Changing Size of Waybar and Applications: How can I change the size of Waybar? Additionally, 

https://github.com/prasanthrangan/hyprdots/issues/1775#issuecomment-2308599454


> Additionally, I would like to know how to adjust the size settings for applications like Firefox.

wdym ? scaling? add the monitor scaling to ` ~/.config./hypr/userprefs.conf` 

https://wiki.hyprland.org/Configuring/Monitors/

> Improving Monitor Refresh Rate: My second monitor has a relatively low refresh rate. Could someone guide me on how to improve or fix this issue?


https://wiki.hyprland.org/Configuring/Monitors/

> Keyboard Layout and Bindings: I need to configure both US and RU keyboard layouts and add custom key bindings. I tried to change this through the hyprland config but was unsuccessful. Could you provide guidance on the correct approach or steps to follow?



https://wiki.hyprland.org/Configuring/Variables/#input
![image](https://github.com/user-attachments/assets/47d4d07f-bd3a-43dc-8054-cc974516be5d)

> Binding Space to Alt Gr: Is there a way to bind the Space key to function as Alt Gr? I would appreciate any guidance on how to set this up.

https://wiki.hyprland.org/Configuring/Uncommon-tips--tricks/#switchable-keyboard-layouts
https://wiki.hyprland.org/Configuring/Uncommon-tips--tricks/#remap-caps-lock-to-ctrl

**Reply by 0x1holon** [reply]
Hi kRHYME7,  thank you for your response.

I’m facing a couple of issues with Waybar and monitor configuration:
Waybar Issue

I receive the following error message:


```
~/.config/waybar/config.ctl
fish: Unknown command. '/home/holon/.config/waybar/config.ctl' exists but is not an executable file.
```

It seems that Fish is trying to execute a configuration file as if it were a command. I believe this is due to an incorrect command entry or misunderstanding of file usage.
Monitor Configuration Issue

When I run hyprctl monitors, I get the following output:


```
Monitor eDP-1 (ID 0):
	1920x1080@120.00200 at 0x0
	description: AU Optronics 0x479D
	make: AU Optronics
	model: 0x479D
	serial: 
	active workspace: 1 (1)
	special workspace: 0 ()
	reserved: 0 32 0 0
	scale: 1.00
	transform: 0
	focused: yes
	dpmsStatus: 1
	vrr: 0
	activelyTearing: false
	disabled: false
	currentFormat: XRGB8888
	availableModes: 1920x1080@120.00Hz 1920x1080@48.00Hz 

Monitor DP-1 (ID 1):
	2560x1440@74.97100 at 1920x0
	description: LG Electronics LG HDR QHD 110NTBKCU589
	make: LG Electronics
	model: LG HDR QHD
	serial: 110NTBKCU589
	active workspace: 2 (2)
	special workspace: 0 ()
	reserved: 0 32 0 0
	scale: 1.60
	transform: 0
	focused: no
	dpmsStatus: 1
	vrr: 0
	activelyTearing: false
	disabled: false
	currentFormat: XRGB8888
	availableModes: 2560x1440@59.95Hz 2560x1440@74.97Hz 1920x1080@74.91Hz 1920x1080@60.00Hz 1920x1080@59.94Hz 1920x1080@50.00Hz 1920x1080@29.97Hz 1680x1050@59.95Hz 1600x900@60.00Hz 1280x1024@75.03Hz 1280x1024@60.02Hz 1280x800@59.81Hz 1152x864@59.96Hz 1280x720@60.00Hz 1280x720@59.94Hz 1280x720@50.00Hz 1024x768@75.03Hz 1024x768@60.00Hz 800x600@75.00Hz 800x600@60.32Hz 720x576@50.00Hz 720x480@59.94Hz 640x480@75.00Hz 640x480@59.94Hz 640x480@59.93Hz 


```
I edited monitors.conf with the following settings:

```
monitor = eDP-1, 1920x1080@120, 0x0, 1
monitor = DP-1, 2560x1440@74.97100, 1920x0,1.6
```
However, I’m still facing issues with the refresh rate (Hz).

Layout Configuration:
I’m also looking for guidance on how to correctly configure layouts. I found some information on the wiki, but I’m not sure how to apply it to my setup. How do I properly configure the layout settings to ensure everything aligns correctly and functions as expected? 

 Waybar Configuration: How do I correctly set up the Waybar configuration to avoid the execution error? I believe I need to place the configuration file in the correct directory with the proper name and format. 

Monitor Configuration: How can I resolve the refresh rate issue with the monitors? Is there a specific way to ensure that the settings in monitors.conf are applied correctly?

Swap Function: I didn’t find information on where to write a swap function. Could you provide guidance on where and how to implement this?

Thank you for your assistance!

**Reply by kRHYME7** [reply]
> It seems that Fish is trying to execute a configuration file as if it were a command. I believe this is due to an incorrect command entry or misunderstanding of file usage.

It is a file where you should edit the layout.  
So what I really mean on that image is to run ` Hyde waybar size 30 ` (30 being the size).  that command will modify the control file 

` ~/.config/waybar/config.ctl ` 


 ******* 
However, I might be very slow to respond, so I would like you to move this query to our discord server. Thanks! These are just the basics, so they should aid you in there. 
https://discord.com/invite/qWehcFJxPa

I will try to update some parts of the wiki soon as a response. (maybe a couple days later)

**Reply by 0x1holon** [reply]
I asked in Dc but only got this answer
![image](https://github.com/user-attachments/assets/76bbc8fc-b167-44eb-848c-7e230754f3fb)
I fixed issue with Waybar, thanks
However, I’m still facing issues with the refresh rate (Hz). 
I also found out that the fps seems to be normal when nothing is open, but as soon as I open any application, the fps becomes small 

my laptop: asus tug gaming f17 (2021) with gtx 1650ti and i7 intel 10th generation, so this cannot be a hardware problem, everything is also well connected and works fine on windows
other issues still there
